src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -439,7 +439,7 @@ private String getInfo(ContentFilter filter) {
                         .append(" arg[")
                         .append(count++)
                         .append("]: `")
-                        .append(Markdown.escapeBacktick(ContentFilter.filter(filter, arg)))
+                        .append(Markdown.escapeBacktick(filter != null ? ContentFilter.filter(filter, arg) : arg))
```

Comment:
previous discussion in https://github.com/jenkinsci/support-core-plugin/pull/522#discussion_r1521707377